talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -118,8 +130,19 @@ pipeline {
    * environment
    */
   environment {
-    MAVEN_SETTINGS = "${SETTINGS_PATH}"
-    DECRYPTER_ARG = "-Dtalend.maven.decrypter.m2.location=${env.WORKSPACE}/.jenkins/"
+    APP_ID = '579232'
+    TALEND_REGISTRY = "artifactory.datapwn.com"
+    TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX = "artifactory.datapwn.com/docker-io-remote/"
+
+    String workspace = "${WORKSPACE}"
```

Comment:
Would you agree to use directly WORKSPACE instead of an medium variable String ?
It would be more consistent with rest of the file and the variable does not simplify current code.